src/js/containers/search/filters/AgencyListContainer.jsx

```diff
@@ -47,10 +47,10 @@ export default class AgencyListContainer extends React.Component {
             results.forEach((item) => {
                 let subAbbreviation = '';
                 let topAbbreviation = '';
-                if (item.subtier_agency.abbreviation !== '') {
+                if (item.subtier_agency.abbreviation !== '' && item.subtier_agency.abbreviation !== null) {
```

Comment:
Isn't this null check unnecessary given our null check in the models?

Also, we could just do a falsey check as follows:

```javascript
if (!obj.property) `
```
which should handle these two conditions in one expression.

Reply:
> Believe it or not, this code isn't run through a model.

I will make it falsy.  I was just following some code I saw elsewhere.